libcontainer/configs/config.go

```diff
@@ -332,6 +332,19 @@ const (
 	Poststop HookName = "poststop"
 )
 
+// HasHook checks if config has any hooks with any given names configured.
```

Comment:
I think we don’t have libct API document at this time. Otherwise this useful API should be mentioned in the doc.